addons/campsite/helper/fn_isInsideCampingArea.sqf

```diff
@@ -0,0 +1,41 @@
+#include "script_component.hpp"
+/*
+ * Author: Gilles
+ *
+ * TODO DOCSTRING
+ *
+ * Arguments:
+ * 0: xyz
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ *
+ * Public: No
+ *
+ * Scope: Client
+ */
+
+// Maybe consider checking this variable 1 time each session?
+scopeName "main";
+
+private _unitLeader = leader player;
+private _characterName = _unitLeader getVariable ["coopr_character_name", ""];
+_characterName = _characterName splitString " ";
+
+while{_characterName find "" > -1} do{
+	_characterName deleteAT (_characterName find " ");
+};
+_characterName = _characterName joinString "";
+private _stringToSearch = format["%1"+"_campingArea_", _characterName];
+private _allMarkers = allMapMarkers select {_x find _stringToSearch == 0};
```

Comment:
rename to `_allMarkersSetByPlayer` just to be more clear for future contributors